deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -448,6 +448,7 @@ export class StackComposer {
                 stackName: `OSMigrations-${stage}-${region}-ReindexFromSnapshot`,
                 description: "This stack contains resources to assist migrating historical data, via Reindex from Snapshot, to a target cluster",
                 stage: stage,
+                otelCollectorEnabled: otelCollectorEnabled,
```

Comment:
Could you do a quick find / replace to make sure this parameter is required? 

`readonly otelCollectorEnabled?:` -> `readonly otelCollectorEnabled:`